clang/test/CodeGenHLSL/builtins/countbits.hlsl

```diff
@@ -0,0 +1,80 @@
+// RUN: %clang_cc1 -std=hlsl2021 -finclude-default-header -x hlsl -triple \
+// RUN:   dxil-pc-shadermodel6.3-library %s -fnative-half-type \
+// RUN:   -emit-llvm -disable-llvm-passes -O3 -o - | FileCheck %s
+
+#ifdef __HLSL_ENABLE_16_BIT
+// CHECK: define noundef i16 @
```

Comment:
i dunno if the `// CHECK: define` add any value stopping at the `@`. I think before we were using them to check the function name, but that can be better done with a `CHECK-LABEL`.